### PR TITLE
Fix travis builds by pinning glide to v0.12.3 

### PR DESCRIPTION
**Pull request by prateek**
details here: #224 

/cc @cw9

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -93,7 +93,7 @@ install-proto-bin: install-vendor
 	go install $(m3db_package)/$(vendor_prefix)/$(protoc_go_package)
 
 install-glide:
-	@which glide > /dev/null || go get -u github.com/Masterminds/glide
+	@which glide > /dev/null || mkdir -p $(GOPATH)/src/github.com/Masterminds && cd $(GOPATH)/src/github.com/Masterminds && git clone https://github.com/Masterminds/glide.git && cd glide && git checkout v0.12.3 && go install
```
Comment (cw9) [inline Makefile]: any particular reason to prefer git clone than go get here?

Comment (robskillington) [inline Makefile]: Yeah have a feeling we should use `go get` and then just cd, checkout and install.  `go get` will check it out as a git repository (or whatever repository it ends up being).

Comment (prateek) [inline Makefile]: Sounds good. Making new version.

Comment (prateek) [inline Makefile]: @cw9 @robskillington done

**Pull request merged**